projects/luarocks.org/package.yml

```diff
@@ -22,6 +22,7 @@ build:
   script: |
     ./configure $ARGS
     make --jobs {{ hw.concurrency }} install
+    sed -i.bak "s_{{prefix}}_\$(cd \$(dirname \$0)/.. \&\& pwd)_g"
```

Comment:
you need to look at the files in `{{prefix}}/bin` to see if any of them need this (this invocation isn't modifying any files). this also wouldn't be a proper fix for a shebang; that should use our script.

Reply:
```shell
jacob@illithid pantry % rg jacob ~/.pkgx/luarocks.org/v3.9.2/bin 
/Users/jacob/.pkgx/luarocks.org/v3.9.2/bin/luarocks-admin
1:#!/Users/jacob/.pkgx/lua.org/v5.4.6/bin/lua
2:package.loaded["luarocks.core.hardcoded"] = { FORCE_CONFIG = true, SYSCONFDIR = [[/Users/jacob/.pkgx/luarocks.org/v3.9.2/etc/luarocks]] }
3:package.path=[[/Users/jacob/.pkgx/luarocks.org/v3.9.2/share/lua/5.4/?.lua;]] .. package.path
4:local list = package.searchers or package.loaders; table.insert(list, 1, function(name) if name:match("^luarocks%.") then return loadfile([[/Users/jacob/.pkgx/luarocks.org/v3.9.2/share/lua/5.4/]] .. name:gsub([[%.]], [[/]]) .. [[.lua]]) end end)

/Users/jacob/.pkgx/luarocks.org/v3.9.2/bin/luarocks
1:#!/Users/jacob/.pkgx/lua.org/v5.4.6/bin/lua
2:package.loaded["luarocks.core.hardcoded"] = { FORCE_CONFIG = true, SYSCONFDIR = [[/Users/jacob/.pkgx/luarocks.org/v3.9.2/etc/luarocks]] }
3:package.path=[[/Users/jacob/.pkgx/luarocks.org/v3.9.2/share/lua/5.4/?.lua;]] .. package.path
4:local list = package.searchers or package.loaders; table.insert(list, 1, function(name) if name:match("^luarocks%.") then return loadfile([[/Users/jacob/.pkgx/luarocks.org/v3.9.2/share/lua/5.4/]] .. name:gsub([[%.]], [[/]]) .. [[.lua]]) end end)
```

so, the shebangs need to be fied, as well as the path references; unfortunately, that'll have to be done using `lua` syntax. let me see if chatgpt knows it. ;)

Reply:
chatgpt thinks this might do it:
```yaml
  - run: |
      sed -i.bak \
          -e 's|{{prefix}}|\" .. debug.getinfo(1).source:match("@?(.*/)") .. \"|g' \
          luarocks-admin luarocks
          rm luarocks-admin.bak luarocks.bak
    working-directory: ${{prefix}}/bin
```

Reply:
ok, think i got it.

Reply:
yup, just had to learn _very very little_ about lua syntax.